src/components/pages/Navbar/NavbarFeatures/LoginButton.js

```diff
@@ -3,15 +3,17 @@ import { useAuth0 } from '@auth0/auth0-react';
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
 
 function LoginButton() {
-  const { loginWithRedirect } = useAuth0();
+  const { loginWithRedirect, isAuthenticated } = useAuth0();
 
   return (
-    <NavBtnLink
-      className="btn btn-primary btn-block"
-      onClick={() => loginWithRedirect()}
-    >
-      Login
-    </NavBtnLink>
+    !isAuthenticated && (
```

Comment:
so simple to use this logic in each of the components. This will clean up the look so much!